### DISCUSSION TITLE
Add support for brotli Content-Encoding

### DISCUSSION THREAD

**Discussion by vladi1X** [Kong/kong#7882] [Help] [answered]
Hello guys,

as Content-Encoding with brotli is becoming more standardized in IT environments and browsers, I wanted to ask if it's planned to implement brotli-compression in Kong.
I think it would be very benefitial for different use cases and users of kong since gzip is also supported. If brotli would also be delivered within the kong-release, it would eliminiate extra steps for the users to compile kong by themselves after every kong release.

Thanks!

**Comment by aparra** [comment, marked as answer]
Kong supports brotli in Gateway 3.6 https://konghq.com/blog/product-releases/kong-gateway-3-6?_ga=2.40925413.635238689.1718019262-2124671994.1663320766
